heka/sandbox/encoders/http_edge_encoder.lua

```diff
@@ -68,24 +61,25 @@ local ns_config = {
 }
 
 function process_message()
-    -- Carry forward payload some incoming fields
```

Comment:
We could stop adding dynamic fields at this point - we could simply add a "PathComponents" array regardless of namespace config, and extract those dynamic fields downstream. I don't know if it's desirable to reference (or split) the namespace configuration in multiple places though.